deploy/config.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: 3scale-istio-adapter-conf
+data:
+  log_json: "true"
```

Comment:
Why not use the dot as a namespace separator just like other config settings below?

Reply:
well, these are global setting but i can change them if you wish

Reply:
that's a valid point, let's keep it that way